This patch is intended to fix reboot.h implict declaration issue when compile on Alpine Linux

Based on : https://github.com/RIPE-NCC/ripe-atlas-software-probe/pull/118

diff --git a/probe-busybox/networking/telnetd.c b/probe-busybox/networking/telnetd.c
index 543de5a..8a67651 100644
--- a/probe-busybox/networking/telnetd.c
+++ b/probe-busybox/networking/telnetd.c
@@ -134,6 +134,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <linux/reboot.h>
+#include <sys/reboot.h>
 
 #define LOGIN_PREFIX	"Atlas probe, see http://atlas.ripe.net/\r\n\r\n"
 #define LOGIN_PROMPT	" login: "
